lms/djangoapps/badges/migrations/0002_data__migrate_assertions.py

```diff
@@ -43,14 +45,20 @@ def forwards(apps, schema_editor):
             data = badge.data
         else:
             data = json.dumps(badge.data)
-        BadgeAssertion(
+        assertion = BadgeAssertion(
             user_id=badge.user_id,
             badge_class=classes[(badge.course_id, badge.mode)],
             data=data,
             backend='BadgrBackend',
             image_url=badge.data['image'],
             assertion_url=badge.data['json']['id'],
-        ).save()
+        )
+        assertion.save()
+        # Would be overwritten by the first save.
```

Comment:
FYI, per Django docs, another option "If you want to be able to modify this field, set the following instead of `auto_now_add=True`: `default=timezone.now` - from `django.utils.timezone.now()`"